compiler/resolution/functionResolution.cpp

```diff
@@ -6224,6 +6224,26 @@ FnSymbol* findCopyInit(AggregateType* at) {
   return ret;
 }
 
+FnSymbol* findAssign(AggregateType* at) {
```

Comment:
I suggest renaming these to `findAssignFn` and `findZeroArgInitFn`, to make clear(er) we are looking for a FnSymbol and not for a SymExpr (of some Symbol) in a call to `==`.